sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.            "SupportedRealtimeInferenceInstanceTypes": [
</pre><p>This list of instance types is key for the inference recommender feature. Add a few sentences explaining how a customer could generate this list of instances that are worth testing. Is there a certain size or type that are best for different tasks, frameworks, data types, data volumes, etc? Maybe there are some rules of thumb that can be shared. If the customer provides a bad list of instances, then they will get bad results from the inference recommender.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Added detailed explanation.